app/src/components/TableList/TableList.scss

```diff
@@ -47,6 +47,10 @@
       color: $negative-red;
     }
 
+    &.-error {
+      color: red;
```

Comment:
Please use $negative-red for color scheme consistency.

Reply:
Will remove when I can.